sidebars.js

```diff
@@ -62,19 +62,15 @@ const sidebars = {
     'dapps/launch-your-ethereum-dapp',
     {
       type: 'category',
-      label: 'Smart Contract',
+      label: 'Developer Toolchains',
       collapsed: false,
       items: [
-        'dapps/smart-contracts/README',
-        'dapps/smart-contracts/create-erc-20-token-on-avalanche-c-chain',
-        'dapps/smart-contracts/deploy-a-smart-contract-on-avalanche-using-remix-and-metamask',
-        'dapps/smart-contracts/using-foundry-with-the-avalanche-c-chain',
-        'dapps/smart-contracts/using-hardhat-with-the-avalanche-c-chain',
-        'dapps/smart-contracts/using-truffle-with-the-avalanche-c-chain',
-        'dapps/smart-contracts/verify-smart-contract-using-hardhat-and-snowtrace',
-        'dapps/smart-contracts/verify-smart-contracts-with-truffle-verify',
-        'dapps/smart-contracts/verify-smart-contracts',
-        'dapps/smart-contracts/add-avalanche-programmatically',
+        'dapps/developer-toolchains/README',
+        'dapps/developer-toolchains/using-foundry-with-the-avalanche-c-chain',
+        'dapps/developer-toolchains/using-hardhat-with-the-avalanche-c-chain',
+        'dapps/developer-toolchains/using-truffle-with-the-avalanche-c-chain',
+        'dapps/developer-toolchains/verify-smart-contract-using-hardhat-and-snowtrace',
+        'dapps/developer-toolchains/verify-smart-contracts-with-truffle-verify',
```

Comment:
As these files' locations have been changed, we would need to add a new entry in https://github.com/ava-labs/avalanche-docs/blob/master/static/_redirects for each of them. For example:

`/dapps/smart-contracts/using-foundry-with-the-avalanche-c-chain /dapps/developer-toolchains/using-foundry-with-the-avalanche-c-chain 301`

Otherwise, people who has previous URL will get an error message. Please add a redirect for each file whose location has changed.